### PR TITLE
[Term Entry] PHP Arrays array_pop()_07/02/2023_9:34:PM

### LINKED ISSUE #2628: [Term Entry] PHP Arrays array_pop()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `array_pop()` function.

The entry should go in a new file in `docs/content/php/concepts/arrays/terms/array-pop/array-pop.md` (Note the dashes rather than underscores)

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by phuongtrieu97coder**
### Description
- Added array_pop() term for PHP
- Closes #2628 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-pop/array-pop.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+Title: 'array_pop()'
+Description: 'Removes the last element of an array, and returns the new number of element values in the array. If the specified array is empty, null will be returned.'
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-pop/array-pop.md:3]:
```suggestion
Description: 'Removes the last element of an array.'
```

File: content/php/concepts/arrays/terms/array-pop/array-pop.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-pop/array-pop.md:7]:
```suggestion
```

Comment (caupolicandiaz) [reply content/php/concepts/arrays/terms/array-pop/array-pop.md:7]: we limit subjects to two items

File: content/php/concepts/arrays/terms/array-pop/array-pop.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+array_pop($array)
+```
+
+The `array_pop()` function has one required parameter:
+
+- `$array`: Specifies the input `array`.
+
+## Example
+
+The following example uses the `array_pop()` function to remove `"Mexico"` element value from the indexed array `$countries`:
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-pop/array-pop.md:30]:
```suggestion
The following example uses the `array_pop()` function to remove `"Mexico"` from the array `$countries`:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@phuongtrieu97coder I've added a few comments. Please update the entry, and tag me when you've finished making the changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@phuongtrieu97coder I made a couple of additional edits. This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@phuongtrieu97coder Looks good to merge.

**Comment by SSwiniarski** [comment]
@phuongtrieu97coder Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/arrays/array-pop

image

**Pull request merged**